tests/devices.spec.ts

```diff
@@ -106,3 +107,63 @@ test("profile edit networks", async ({ page }) => {
 
   await deleteProfile(page, profile);
 });
+
+test("add, edit and remove a proxy device", async ({ page }) => {
+  const profile = randomProfileName();
+
+  //Add Proxy Device
+  await startProfileCreation(page, profile);
+  await page.getByText("Proxy", { exact: true }).click();
+  await page.getByRole("button", { name: "New Proxy Device" }).click();
+  await page.getByLabel("Bind").selectOption({ index: 2 });
+  await page.getByLabel("*Address").first().fill("127.0.0.1");
+  await page.getByLabel("*Port").first().fill("3000");
+  await page.getByLabel("*Address").nth(1).fill("127.0.0.2");
+  await page.getByLabel("*Port").nth(1).fill("3001");
+  await finishProfileCreation(page, profile);
+
+  //Edit and save Proxy Device
+  await visitProfile(page, profile);
+  await page.getByTestId("tab-link-Configuration").click();
+  await page.getByText("Proxy", { exact: true }).click();
+  await page.getByLabel("*Address").first().fill("127.0.0.3");
+  await saveProfile(page, profile, 1);
+
+  //Edit and save Proxy Device (Refresh)
+  await visitProfile(page, profile);
+  await page.getByTestId("tab-link-Configuration").click();
+  await page.getByText("Proxy", { exact: true }).click();
+  await expect(
+    page
+      .getByRole("gridcell", { name: "127.0.0.3" })
+      .getByPlaceholder("127.0.0.1"),
```

Comment:
I think we should work with the label instead of the placeholder here and also in the other assertions below.